docs/website/sidebars.json

```diff
@@ -1,6 +1,6 @@
 {
   "users": {
-    "General": ["users/general/getting-started", "users/general/settings"]
+    "General": ["users/general/getting-started", "users/general/settings", "users/general/publish", "users/general/creating-campaigns"]
```

Comment:
I'd put these in a separate Content Management (or something similar) section, and just call them `Content` / `Campaigns`, or add 2 sections for them.

- Managing Content
  - Content
  - Campaigns

or:

- Content
  - Editing
- Campaigns
  - Editing